docs/build-instructions/os-x.md

```diff
@@ -8,11 +8,22 @@
 
 ## Instructions
 
-  ```sh
-  git clone https://github.com/atom/atom.git
-  cd atom
-  script/build # Creates application at /Applications/Atom.app
-  ```
+  If you have problems with permissions don't forget to prefix with `sudo`
```

Comment:
No, we specifically don't want them to use sudo. Everything should work without using it.